documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -28,96 +28,100 @@ We request OpenAPI(Swagger) spec authoring be assigned to engineers who have an
-| [R2008](#R2008)	| [MutabilityWithReadOnlyRule](#R2008) | When property is modeled as "readOnly": true then x-ms-mutability extension can only have "read" value. When property is modeled as "readOnly": false then applying x-ms-mutability extension with only "read" value is not allowed. Extension contains invalid values: '{0}'. | Error |
-| [R2025](#R2025)	| [NextLinkPropertyMustExist](#R2025)	| The property '{0}' specified by nextLinkName does not exist in the 200 response schema. Please, specify the name of the property that provides the nextLink. If the model does not have the nextLink property then specify null. | Error |
-| [R2028](#R2028) | [NonEmptyClientName](#R2028) | Empty x-ms-client-name property | Error |
-| [R2060](#R2060) | [PageableRequires200Response](#R2060) | A response for the 200 HTTP status code must be defined to use x-ms-pageable | Error |
-| M2019	| ResourceIsMsResourceValidation | A 'Resource' definition must have x-ms-azure-resource extension enabled and set to true. |	Error |
-| [R2058](#R2058) |[XmsPathsMustOverloadPaths](#R2058) | Paths in `x-ms-paths` must overload a normal path in the paths section, i.e. a path in the `x-ms-paths` must either be same as a path in the paths section or a path in the paths sections followed by additional parameters. | Error |
-| [R2012](#R2012)	| [XmsClientNameParameter](#R2012) | Value of 'x-ms-client-name' cannot be the same as '{0}' Property/Model. | Error |
-| [R2013](#R2013)	| [XmsClientNameProperty](#R2013) | Value of 'x-ms-client-name' cannot be the same as '{0}' Property/Model. | Error |
-| M2047	| ParameterNameValidation | Parameter Must have the "name" property defined with non-empty string as its value | Error |
-| [R2056](#R2056)	| [RequiredReadOnlyProperties](#R2056) | Property '{0}' is a required property. It should not be marked as 'readonly'. | Error |
-| M2054	| SecurityDefinitionsStructureValidation | An OpenAPI(swagger) spec must have security definitions and must adhere to the specific structure. | Error |
-| [R2006](#R2006)	| [ControlCharactersNotAllowed](#R2006) | Specification must not contain any control characters. | Error |
-| [R2009](#R2009)	| [ArraySchemaMustHaveItems](#R2009) | A property of type `Array` must have `items` defined in its `Schema`. | Error |
-| [R2022](#R2022) | [XmsExamplesRequired](#R2022) | Please provide x-ms-examples describing minimum/maximum property set for response/request payloads for operations.{0} | Error |
+| Id | Rule Name |
+| --- | --- |
+| [R2024](#R2024) | [AnonymousBodyParameter](#R2024) |
+| [R2026](#R2026) | [AvoidAnonymousTypes](#R2026) |
+| [R2014](#R2014)	| [OperationParametersValidation](#R2014)	|
+| [R2027](#R2027)	| [DefaultMustBeInEnum](#R2027) |
+| M1004	| ListByOperationsValidation |
+| [R1001](#R1001)	| [OperationIdNounInVerb](#R1001)	|
+| [R2055](#R2055)	| [OneUnderscoreInOperationId](#R2055) |
+| [R2014](#R2014)	| [ServiceDefinitionParameters](#R2014) |
```

Comment:
Please remove line for ServiceDefinitionParamaters, rule has been renamed to "ParameterNotDefinedInGlobalParameters" and it's already covered in the doc.